ipaclient/install/client.py

```diff
@@ -1409,21 +1414,30 @@ def get_local_ipaddresses(iface=None):
     return ips
 
 
-def do_nsupdate(update_txt):
+def do_nsupdate(update_txt, options, server):
```

Comment:
we have to add explicitly 'server host port' before the update text to address the issue in https://gitlab.isc.org/isc-projects/bind9/-/merge_requests/9797. We don't pass any configuration like that in `do_nsupdate()` callers, so it is safe to prefix `update_txt` with it. See https://github.com/SSSD/sssd/blob/master/src/providers/be_dyndns.c#L465-L477 for details on SSSD side.

Reply:
Ok, this seems to be done in the caller, so this can be ignored.